tests/release/release.go

```diff
@@ -0,0 +1,133 @@
+package release
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	gitopsv1alpha1 "github.com/redhat-appstudio/managed-gitops/appstudio-shared/apis/appstudio.redhat.com/v1alpha1"
+
+	// appstudiov1alpha1 "github.com/redhat-appstudio/release-service/api/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	// klog "k8s.io/klog/v2"
+	"knative.dev/pkg/apis"
+)
+
+var snapshotComponents = []gitopsv1alpha1.ApplicationSnapshotComponent{
+	{"component-1", "quay.io/redhat-appstudio/component1@sha256:d5e85e49c89df42b221d972f5b96c6507a8124717a6e42e83fd3caae1031d514"},
+	{"component-2", "quay.io/redhat-appstudio/component2@sha256:a01dfd18cf8ca8b68770b09a9b6af0fd7c6d1f8644c7ab97f0e06c34dfc5860e"},
+	{"component-3", "quay.io/redhat-appstudio/component3@sha256:d90a0a33e4c5a1daf5877f8dd989a570bfae4f94211a8143599245e503775b1f"},
+}
+
+var _ = framework.ReleaseSuiteDescribe("test-release-service-happy-path", Label("release"), func() {
+	defer GinkgoRecover()
+	// Initialize the tests controllers
+	framework, err := framework.NewFramework()
+	Expect(err).NotTo(HaveOccurred())
+
+	var devNamespace string
+	var managedNamespace string
+
+	var _ = Describe("test-release-service-happy-path", func() {
+		BeforeAll(func() {
```

Comment:
This is the same exact code as https://github.com/redhat-appstudio/e2e-tests/pull/200/files#diff-45cefccd274b7438a6b13ccdd6e4a7e27936940985bedd59dab8508329725001
Why is it in two different places?

Reply:
Why is this code here when it is a copy of the code in your other PR?